src/rules/noDuplicateVariableRule.ts

```diff
@@ -60,11 +64,12 @@ class NoDuplicateVariableWalker extends Lint.ScopeAwareRuleWalker<ScopeInfo> {
         var propertyName = <ts.Identifier> node.name;
         var variableName = propertyName.text;
         var currentScope = this.getCurrentScope();
+        var currentBlockScope = this.getCurrentBlockScope();
         // determine if the appropriate bit in the parent (VariableDeclarationList) is set, which indicates this is a "let"
         var declarationIsLet = (Math.floor(node.parent.flags / ts.NodeFlags.Let) % 2) === 1;
 
         if (currentScope.varNames.indexOf(variableName) >= 0) {
-            // if there was a previous var declaration with the same name, this declaration is invalid
+            // if there was a previous var declaration with the same name, this declaration is invalid (regardless of let)
```

Comment:
I'm not sure you need this parenthetical, and if you do want it you should probably explain that it'll be picked up later